### PR TITLE
add benchmarks

### PR DESCRIPTION
**Pull request by haifeng-jin**
image

### CODE REVIEW

File: templates/getting_started/benchmark.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+
+\* _LLM inference with the PyTorch backend is abnormally slow at this time
+because KerasNLP uses static sequence padding, unlike HuggingFace. This will be
+addressed soon._
+
+## Discussion
+
+### Key Finding 1: There is no "best" backend
+
+Each of Keras 3's three backends offers unique strengths. Crucially, from a
```
Comment (fchollet) [inline templates/getting_started/benchmark.md:98]: "Each of the three backends of Keras 3 offers"

File: templates/getting_started/benchmark.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+## Conclusions
+
+Framework performance depends heavily on the specific model. Keras 3 empowers
+you to select the fastest framework for your task – an option almost always to
+outperform both Keras 2 and reference PyTorch implementations. Importantly,
+Keras 3 models deliver excellent out-of-the-box performance without requiring
+complex, low-level optimizations.
+
+
+## Reference
```
Comment (fchollet) [inline templates/getting_started/benchmark.md:175]: References

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: templates/getting_started/benchmark.md [hunk 3]
```diff
@@ -0,0 +1,192 @@
+# Keras 3 benchmarks
+We benchmark the three backends of Keras 3
```
[the remark this answers is not in the document]
Comment (haifeng-jin) [reply templates/getting_started/benchmark.md:3]: Done.

File: templates/getting_started/benchmark.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+([TensorFlow](https://tensorflow.org/),
+[JAX](https://jax.readthedocs.io/en/latest/), [PyTorch](https://pytorch.org/))
+against native PyTorch implementations ([HuggingFace](https://huggingface.co/)
```
[the remark this answers is not in the document]
Comment (haifeng-jin) [reply templates/getting_started/benchmark.md]: Done.

File: templates/getting_started/benchmark.md [hunk 5]
```diff
@@ -0,0 +1,192 @@
...
+and [Meta Research](https://github.com/facebookresearch/)) and against Keras 2
```
[the remark this answers is not in the document]
Comment (haifeng-jin) [reply templates/getting_started/benchmark.md]: Done.

File: scripts/master.py [hunk 6]
```diff
@@ -19,6 +19,10 @@
                     "path": "intro_to_keras_for_engineers",
                     "title": "Introduction to Keras for engineers",
                 },
+                {
+                    "path": "benchmark",
```
[the remark this answers is not in the document]
Comment (haifeng-jin) [reply scripts/master.py]: Done.

File: templates/getting_started/benchmark.md (hunk 1 above)
Comment (haifeng-jin) [reply templates/getting_started/benchmark.md]: Done.

File: templates/getting_started/benchmark.md (hunk 2 above)
Comment (haifeng-jin) [reply templates/getting_started/benchmark.md]: Done.

**Pull request merged**